core/admin/EE_Admin_Page_CPT.core.php

```diff
@@ -944,76 +932,6 @@ public function load_global_scripts_styles()
     }
 
 
-    /**
-     * @throws InvalidArgumentException
-     * @throws InvalidDataTypeException
-     * @throws InvalidInterfaceException
-     */
-    private function loadEspressoEditorAssetManager()
-    {
-        EE_Dependency_Map::register_dependencies(
-            'EventEspresso\core\domain\services\assets\EspressoEditorAssetManager',
-            array(
-                'EventEspresso\core\domain\Domain'                   => EE_Dependency_Map::load_from_cache,
-                'EventEspresso\core\services\assets\AssetCollection' => EE_Dependency_Map::load_from_cache,
-                'EventEspresso\core\services\assets\Registry'        => EE_Dependency_Map::load_from_cache,
-            )
-        );
-        $this->asset_manager = $this->loader->getShared(
-            'EventEspresso\core\domain\services\assets\EspressoEditorAssetManager'
-        );
-        add_action('admin_enqueue_scripts', [$this, 'enqueueEspressoEditorAssets'], 100);
-    }
-
-
-    /**
-     * @throws InvalidArgumentException
-     * @throws InvalidDataTypeException
-     * @throws InvalidInterfaceException
-     */
-    private function loadLegacyEditorAssetManager()
-    {
-        EE_Dependency_Map::register_dependencies(
-            'EventEspresso\core\domain\services\assets\LegacyEditorAssetManager',
-            array(
-                'EventEspresso\core\domain\Domain'                   => EE_Dependency_Map::load_from_cache,
-                'EventEspresso\core\services\assets\AssetCollection' => EE_Dependency_Map::load_from_cache,
-                'EventEspresso\core\services\assets\Registry'        => EE_Dependency_Map::load_from_cache,
-                'EE_Currency_Config'                                 => EE_Dependency_Map::load_from_cache,
-            )
-        );
-        $this->asset_manager = $this->loader->getShared(
-            'EventEspresso\core\domain\services\assets\LegacyEditorAssetManager'
-        );
-        add_action('admin_enqueue_scripts', [$this, 'enqueueLegacyEditorAssets'], 100);
-    }
-
-
-    /**
-     * enqueue_scripts - Load the scripts and css
-     *
-     * @return void
-     * @throws DomainException
-     */
-    public function enqueueEspressoEditorAssets()
-    {
-        $this->asset_manager->enqueueBrowserAssets();
-    }
-
-
-    /**
-     * enqueue_scripts - Load the scripts and css
-     *
-     * @return void
-     * @throws DomainException
-     */
-    public function enqueueLegacyEditorAssets()
-    {
-        $this->asset_manager->enqueueBrowserAssets();
-        wp_enqueue_script('ee-accounting');
-    }
-
-
```

Comment:
have no idea what was the purpose of this piece, but seeing code being removed is pleasing, it might mean that some of the obsolete or overengineered